checka11y.css

```diff
@@ -87,3 +87,138 @@ a[target = "_blank"]::after /* Links that open a new page/tab target="_blank" */
   content: " WARNING: Links that open in a new tab or window should be communicated to the user, ensure the user knows of this behavior" !important;
   font-family: var(--checka11y-font, cursive);
 }
+
+a > a {
```

Comment:
I like the CSS you've added, but could we remove the `>`. This is because `>` targets any direct children, but we want to be checking for **any** children/grandchildren etc.

So, we should be ok with:

```css
a a {
  ...
}
```

And similar for the rest. 😊